packages/cli/src/commands/dao_cmds/install.js

```diff
@@ -76,6 +77,16 @@ export const handler = async function ({
     apmOptions.ensRegistryAddress
   )
 
+  const accounts = await web3.eth.getAccounts()
+  const userAddress = accounts[0]
+  if (!userAddress) throw Error(`Error getting accounts from web3`)
```

Comment:
Is there no helper function already existing in this repository for lines 80-82?

For example, in the case, you would like to change this error msg everywhere will it produce more work as required.

Reply:
There is no helper function for this check at the moment (lines 80-82), do you want me to add it to this PR?

Reply:
This is really bad. Thanks for checking. This means we keep it for now as it is and can improve it later on when we care generally about tech-debt here in this project.